game-headed/build.install4j

```diff
@@ -691,6 +690,49 @@ return true;</string>
                 </serializedBean>
                 <condition />
               </action>
+              <action name="" id="458" customizedId="" beanClass="com.install4j.runtime.beans.actions.control.SetVariableAction" enabled="true" commentSet="false" comment="" actionElevationType="inherit" rollbackBarrier="false" rollbackBarrierExitCode="0" multiExec="false" failureStrategy="2" errorMessage="">
+                <serializedBean>
+                  <java class="java.beans.XMLDecoder">
+                    <object class="com.install4j.runtime.beans.actions.control.SetVariableAction">
+                      <void property="script">
+                        <object class="com.install4j.api.beans.ScriptProperty">
+                          <void property="value">
+                            <string>final boolean is32BitJavaRuntime = "x86".equals(System.getProperty("os.arch"));
```

Comment:
I'd leave that to your more than capable hands.
As long as everything works with the Oracle JDK and the openJDK we're probably fine.
I don't think there are a lot of users that will ever be using something else on a 32-bit machine.
OpenJDK could potentially get more important for Windows because of the licensing stuff that oracle has for jdks starting from JDK 11 ( I think)

Reply:
> I'd leave that to your more than capable hands.

Big Grin : )